sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/OrderbyDocumentQueryTest.java

```diff
@@ -28,8 +35,11 @@
 import com.azure.cosmos.implementation.query.QueryItem;
 import com.azure.cosmos.implementation.routing.Range;
 import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
 import io.reactivex.subscribers.TestSubscriber;
 import org.apache.commons.lang3.StringUtils;
+import org.jetbrains.annotations.NotNull;
```

Comment:
I suspect this code won't compile in the CI. 
Change to assertJ import instead intellj/jetbrain assertion.

Reply:
Yup intellj added it :-) It worked on the CI too somehow. But removed it now

Reply:
Good catch :-)